### ISSUE TITLE
seed line should NOT be visible right away

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#124]
```
What steps will reproduce the problem?
1. latest revision offers a neat functionality, the possibility of editing the 
seed line
2. this functionality is useful for testing and has been requested by users
3. however despite being in the game configuration page, it should not be 
visible right away for not always the user needs to change that line and has 
the high likelihood of scaring the user with too much technicality

What is the expected output? What do you see instead?
in my view there should be a nice button that when pushed either shows a popup 
or simply makes visible that line

```

-----
Original issue reported on code.google.com by `vittorio...@gmail.com` on 20 Dec 2010 at 12:02
* Blocking: #115

**Comment by GoogleCodeExporter** [comment]
```
So long as you implement an interface that:
1) notifies people that they are playing the same game or that it is using the 
same random number (maybe an illuminated pair of dice that go dim after one 
game or something)
2) offers opportunity to save/edit seed, whether adding an extra few clicks or 
not, whatever

Right now the current interface offers both.

Another option is to make the default seed shorter and more readable.  Other 
games that offer this same functionality often use alphanumeric seeds, not hex 
ones. Expanding the character set from 16 to 36 allows tightening the seed 
while offering same level of entrophy.

```

-----
Original comment by `kyberneticist@gmail.com` on 20 Dec 2010 at 3:16

**Comment by GoogleCodeExporter** [comment]
```
i agree on 1 and 2.
i just don't think think that interface offers (1) as it's difficult to notice 
changes on a string of random letters and numbers. We could make this more 
noticeable with a brief flash of the hypothetic button.

i have no objections about the second option, it could be a nice addition: one 
could play "famous quotes" from it. Despite that i still think it should be not 
visible in the main interface, like other games that offer a similar 
functionality do. 
```

-----
Original comment by `vittorio...@gmail.com` on 20 Dec 2010 at 3:31

**Comment by GoogleCodeExporter** [comment]
```
fixed in 0.9.15 and in HEAD
```

-----
Original comment by `vittorio...@gmail.com` on 29 Dec 2010 at 3:05
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Verified, button is added.
```

-----
Original comment by `pilarski...@gmail.com` on 29 Nov 2011 at 7:09
* Changed state: **Verified**
